src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb

```diff
@@ -173,6 +170,38 @@ Public MustInherit Class BasicTestBase
             verify)
     End Function
 
+    '' NEW TARGET
```

Comment:
It just to help me find that method, within only of the similar overloads. Ultimately I'd like to remove as many of the XML based ones as possible.